backend/src/routes/group.ts

```diff
@@ -55,7 +50,7 @@ router.put('/member/:grouphandle/:userhandle', async (ctx, next) => {
 		return;
 	}
 	const user = await User.findOneBy({
-		handle: ctx.params.userhandle,
+		handle: ctx.state.user.handle,
```

Comment:
here too, you'll have to use ctx.state.user.id then get the username from the user obj (+ change handle -> username)